test/test_maf.py

```diff
@@ -7,12 +7,12 @@ def test_maf():
     variant_collection_from_maf = load_maf("data/tcga_ov.head.maf")
     expected_variants = [
         Variant(1, 1650797, "A", "G", ensembl),
-        Variant(1, 231401797, "A", "C", ensembl),
         Variant(1, 23836447, "C", "A", ensembl),
+        Variant(1, 231401797, "A", "C", ensembl),
         Variant(11,124617502, "C", "G", ensembl),
     ]
     eq_(len(variant_collection_from_maf), len(expected_variants))
-    for v_expect, v_maf in zip(expected_variants, variant_collection_from_maf):
+    for v_expect, v_maf in zip(expected_variants, sorted(variant_collection_from_maf)):
```

Comment:
Yeah, using a `set` results in an undefined order for iteration.

Agree with your suggestion. Sorting is fast enough that I don't see it mattering to do it every time we iterate through the collection, even for large collections. If we do hit performance issues, may want to move to a sorted set implementation.